config/release/gcr-tekton-dashboard.yaml

```diff
@@ -1,3 +1,54 @@
+# ------------------- Dashboard Service Account ------------------- #
+
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app: tekton-dashboard
+  name: tekton-dashboard
+  namespace: tekton-pipelines
+---
+# ------------------- Dashboard Role & Role Binding ------------------- #
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: tekton-dashboard-minimal
+rules:
+  - apiGroups: [""]
+    resources: ["serviceaccounts"]
+    verbs: ["get", "list", "update"]
```

Comment:
@carlos-logro FYI, when you add your secret patching you'll need to add the `patch` verb here - take note of this role Stephen is adding or you'll run into RBAC issues, I'm cool for the patch verb to be added as part of that functionality you're adding